src/ClusterBootstrap/cloud_init_deploy.py

```diff
@@ -979,7 +985,9 @@ def run_command(args, command, parser):
             if nargs[0] == "servicesprerequisite":
                 push_all_prerequisite_docker_images(args, config)
     if command == "test":
-        print(rs)
+        config = get_stat_of_sku(config)
+        def_val = default_vc_entries(config)
```

Comment:
Is `default_vc_entries` imported?

Reply:
that's for test, I'd delete these lines.